enterprise/api/v1/serializers.py

```diff
@@ -184,6 +184,11 @@ def save(self):  # pylint: disable=arguments-differ
         __, created = models.EnterpriseCourseEnrollment.objects.get_or_create(
             enterprise_customer_user=self.enterprise_customer_user,
             course_id=course_id,
+            defaults={
+                'source': models.EnterpriseEnrollmentSource.get_source(
+                    models.EnterpriseEnrollmentSource.OFFER_REDEMPTION
```

Comment:
Just curious - how did we pick this default?

Reply:
This is less a default and more an indication that if we `create` we will set the `source`.